projects/go-lib/src/lib/components/go-timepicker/go-timepicker.component.ts

```diff
@@ -33,6 +33,14 @@ export class GoTimepickerComponent implements OnInit {
     this.selectedTime = this.changeTimeFormat(this.control.value);
   }
 
+  ngAfterViewInit(): void {
+    this.control.valueChanges.subscribe((value: any) => {
+      if (value) {
```

Comment:
Why are we checking if there is a value? If someone wants to programmatically null out the selectedTime should they be able to?

Reply:
So, Can I remove the condition? Or I will compare the value with proper time format.